### DISCUSSION TITLE
GitHub Pages does not update website

### DISCUSSION THREAD

**Discussion by hirothreading** [community/community#61832] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi everyone

I've set up a personal website using Hugo (papermod theme) and deployed with GitHub Pages and Actions. The repo for the website can be found here:
https://github.com/hirothreading/hirothreading.github.io

The problem is that no matter what I try to do, the "Teaching" page of my website will not update. When I build the site on localhost, the page renders as it should: 
![screen_shot 2023-07-26 at 14 09 54](https://github.com/community/community/assets/28826838/0bfa56b7-557d-4ad2-9edc-7e5f1f095fcd)

However, when I navigate to the website (https://hirothreading.github.io/teaching/), even on completely different machines I only get the following site:
![screen_shot 2023-07-26 at 14 10 51](https://github.com/community/community/assets/28826838/ef0c5153-f1c1-42e6-8c91-a45c6cab08e4)

I'm not sure if this is some kind of bug, but I'm at a loss to finding a solution besides maybe deleting my repo and recreating it. I've tried editing the index.html files on both my machine and the GitHub site to force some kind of commit/reload, but I've been unsuccessful. 

Any ideas? Seems like a very odd and rare issue.

**Comment by Manishak798** [comment]
Hey,
You are getting this issue because you dint uploaded the same files accordingly,  as GitHub pages will only display the site according to the path given in the index.html. Please consider reuploading your files with the same file path as in your local system.

**Reply by hirothreading** [reply]
I'm not sure what you mean. I even tried removing the entire public folder, pushing to GitHub, building with Hugo, and then pushing to GitHub again. Still nothing.

**Reply by hirothreading** [reply]
Also, the site works totally fine on my local machine when I run "hugo server -D"

**Comment by BaraaBasim** [comment]
it can be due to caching, github pages caches your website to improve performance you can try opening your website in incognito. Also check the build logs, using Hugo which is a static site generator can cause underlying problems.

**Reply by hirothreading** [reply]
No, I tried modifying the index.html file to clear caching. It still didn't fix the problem.

Hugo renders my "Research" page and associated index.html file just fine. It's only the "Teaching" page that has a problem.

**Comment by hirothreading** [comment]
Is there a way to force GitHub to clear whatever "cache" or "memory" on its side?
